docs/vectorizer-api-reference.md

```diff
@@ -518,10 +537,35 @@ considerations.
 
 The available functions are:
 
+- [ai.scheduling_default](#aischeduling_default): to use the plwhen you want manual control over when the vectorizer runs. Use this when you're using an external scheduling system, as is the case with self-hosted deployments.
 - [ai.scheduling_none](#aischeduling_none): when you want manual control over when the vectorizer runs. Use this when you're using an external scheduling system, as is the case with self-hosted deployments.
 - [ai.scheduling_timescaledb](#aischeduling_timescaledb): leverages TimescaleDB's robust job scheduling system, which is designed for reliability and scalability. Use this when you're using Timescale Cloud.
 
 
+### ai.scheduling_default
+
+You use `ai.scheduling_default` to use the platform-specific default scheduling configuration.
+
+On Timescale Cloud, the default is `ai.scheduling_timescaledb()`. On self-hosted, the default is `ai.scheduling_none()`.
```

Comment:
Should we explain why it's different in one vs the other?

Reply:
done